### DISCUSSION TITLE
The Runtime Trade-off

### DISCUSSION THREAD

**Discussion by no-1ne** [chakra-ui/chakra-ui#3281] [Q&A]
Hello, thank you for sharing your work, maintaining and further contributions. 

coming to the issue, 
in the URL, https://chakra-ui.com/docs/comparison#the-runtime-trade-off-%EF%B8%8F
There is a trade off mentioned about using css in js. Has anyone overcome it using something like https://github.com/atlassian-labs/compiled or https://github.com/callstack/linaria

**Comment by vasconx** [comment]
I would also like to know what the actual chakra weight is compared to other css systems
